charts/ChartConfig.tsx

```diff
@@ -237,6 +248,13 @@ export class ChartConfig {
             : formatYear
     }
 
+    // returns entities sorted
+    @computed get uniqueEntitiesAcrossDimensions() {
```

Comment:
nit: maybe just call the function `sortedUniqueEntitiesAcrossDimensions` and ditch the line comment. I prefer long concrete symbol names if it means you can drop a comment.

Reply:
Makes sense, will do